doc/flatten.md

```diff
@@ -34,5 +34,11 @@ cli:
        cli-flatten-payload-max-poly-as-resource-prop-count: 8
        # max properties allowed from flatten of sub-class as param
        cli-flatten-payload-max-poly-as-param-prop-count: 8
+       # if you want to keep the flattened models even if they are not used
+       # default: false
+       cli-flatten-keep-unused-flattened-models: false
+       # setting this to false will skip parameter flattening for operations that have multiple requests (ie, JSON and BINARY)
```

Comment:
>setting this to false will skip parameter flattening for operations that have multiple requests (ie, JSON and BINARY) [](start = 9, length = 117)

what's this for?

Reply:
It is a option in modelerfour, `multiple-request-parameter-flattening` https://github.com/Azure/autorest.modelerfour/blob/master/modelerfour/readme.md#modelfour-options
It decides if one request has multiple parameter, should we still do flatten. By default it's false. It means if payload has more than one parameter, we will skip flatten